## `nixpkgs-review` result

Generated using [`nixpkgs-review-gha`](https://github.com/Defelo/nixpkgs-review-gha)

Command: `nixpkgs-review pr 445168`
Commit: [`0169c4fe8f36cb577a31bc6b4fc25f563aa3215d`](https://github.com/NixOS/nixpkgs/commit/0169c4fe8f36cb577a31bc6b4fc25f563aa3215d) ([subsequent changes](https://github.com/NixOS/nixpkgs/compare/0169c4fe8f36cb577a31bc6b4fc25f563aa3215d..pull/445168/head))
Merge: [`4d68c7a8f33fcc567079296613b94d8d7a8f0ccc`](https://github.com/NixOS/nixpkgs/commit/4d68c7a8f33fcc567079296613b94d8d7a8f0ccc)

Logs: https://github.com/UlyssesZh/nixpkgs-review-gha/actions/runs/17968566124


---
### `x86_64-linux`
<details>
  <summary>:fast_forward: 10 packages marked as broken and skipped:</summary>
  <ul>
    <li>beekeeper-studio</li>
    <li>bitcomet</li>
    <li>deltatouch</li>
    <li>gfie</li>
    <li>googleearth-pro</li>
    <li>holochain-launcher</li>
    <li>lomiri.morph-browser</li>
    <li>lomiri.morph-browser.doc</li>
    <li>runescape</li>
    <li>tonelib-zoom</li>
  </ul>
</details>
<details>
  <summary>:x: 21 packages failed to build:</summary>
  <ul>
    <li>ciscoPacketTracer7</li>
    <li>ciscoPacketTracer8</li>
    <li>hqplayer-desktop</li>
    <li>hqplayer-desktop.doc</li>
    <li>ideamaker</li>
    <li>jigasi</li>
    <li>mblock-mlink</li>
    <li>mullvad-vpn</li>
    <li>odafileconverter</li>
    <li>odoo16</li>
    <li>odoo16.dist</li>
    <li>odoo17</li>
    <li>odoo17.dist</li>
    <li>tests.buildFHSEnv.liblzma</li>
    <li>tests.buildFHSEnv.libtinfo</li>
    <li>tests.hardeningFlags-clang.allExplicitDisabledPie</li>
    <li>tests.hardeningFlags-clang.pieExplicitDisabled</li>
    <li>upwork</li>
    <li>wechat-uos</li>
    <li>wonderdraft</li>
    <li>xunlei-uos</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 364 packages built:</summary>
  <ul>
    <li>airtame</li>
    <li>aja-desktop-software</li>
    <li>alfaview</li>
    <li>amdenc</li>
    <li>amf</li>
    <li>android-studio-for-platform (androidStudioForPlatformPackages.stable)</li>
    <li>androidStudioForPlatformPackages.canary</li>
    <li>ipscan (angryipscanner)</li>
    <li>apkg</li>
    <li>apkg.dist</li>
    <li>appgate-sdp</li>
    <li>apt</li>
    <li>apt.dev</li>
    <li>apt.doc</li>
    <li>apt.man</li>
    <li>aptakube</li>
    <li>aws-workspaces</li>
    <li>ayatana-indicator-bluetooth</li>
    <li>ayatana-indicator-datetime</li>
    <li>ayatana-indicator-display</li>
    <li>ayatana-indicator-power</li>
    <li>ayatana-indicator-session</li>
    <li>ayatana-indicator-sound</li>
    <li>bdf2psf</li>
    <li>beyond-identity</li>
    <li>bilibili</li>
    <li>bisq2</li>
    <li>bitscope.chart</li>
    <li>bitscope.console</li>
    <li>bitscope.display</li>
    <li>bitscope.dso</li>
    <li>bitscope.logic</li>
    <li>bitscope.meter</li>
    <li>bitscope.proto</li>
    <li>bitscope.server</li>
    <li>bitwig-studio (bitwig-studio5)</li>
    <li>bitwig-studio3</li>
    <li>bitwig-studio4</li>
    <li>bitwig-studio5-unwrapped</li>
    <li>blackfire</li>
    <li>brave</li>
    <li>breitbandmessung</li>
    <li>cargo-deb</li>
    <li>cato-client</li>
    <li>cider-2</li>
    <li>cloudflare-warp</li>
    <li>cozette</li>
    <li>cups-brother-dcp1610wlpr</li>
    <li>cups-brother-dcpl3550cdw</li>
    <li>cups-brother-dcpt310</li>
    <li>cups-brother-dcpt725dw</li>
    <li>cups-brother-hl1110</li>
    <li>cups-brother-hl1210w</li>
    <li>cups-brother-hl2260d</li>
    <li>cups-brother-hl3140cw</li>
    <li>cups-brother-hl3170cdw</li>
    <li>cups-brother-hll2340dw</li>
    <li>cups-brother-hll2350dw</li>
    <li>cups-brother-hll2375dw</li>
    <li>cups-brother-hll3230cdw</li>
    <li>cups-brother-mfcl2750dw</li>
    <li>cups-brother-mfcl2800dw</li>
    <li>dcp375cw-cupswrapper</li>
    <li>dcp375cwlpr</li>
    <li>dcp9020cdw-cupswrapper</li>
    <li>dcp9020cdwlpr</li>
    <li>debian-devscripts</li>
    <li>debootstrap</li>
    <li>debos</li>
    <li>deconz</li>
    <li>deezer-enhanced</li>
    <li>dell-command-configure</li>
    <li>distrobuilder</li>
    <li>dpkg</li>
    <li>easyroam-connect-desktop</li>
    <li>edgetpu-compiler</li>
    <li>eigenwallet</li>
    <li>enpass</li>
    <li>epson-alc1100</li>
    <li>eudic</li>
    <li>expressvpn</li>
    <li>feishu</li>
    <li>fflinuxprint</li>
    <li>flashprint</li>
    <li>fxlinuxprint</li>
    <li>fxlinuxprintutil</li>
    <li>gamepad-tool</li>
    <li>git-buildpackage</li>
    <li>git-buildpackage.dist</li>
    <li>gohufont</li>
    <li>greenfoot</li>
    <li>guymager</li>
    <li>hakuneko</li>
    <li>headset</li>
    <li>hydra</li>
    <li>hydra.doc</li>
    <li>insync</li>
    <li>insync-nautilus</li>
    <li>intune-portal</li>
    <li>jibri</li>
    <li>jicofo</li>
    <li>jitsi-meet-prosody</li>
    <li>jitsi-videobridge</li>
    <li>keeweb</li>
    <li>kodelife</li>
    <li>koreader</li>
    <li>lexmark-aex</li>
    <li>libayatana-common</li>
    <li>lightworks</li>
    <li>lomiri.libusermetrics</li>
    <li>lomiri.libusermetrics.dev</li>
    <li>lomiri.libusermetrics.doc</li>
    <li>lomiri.lomiri</li>
    <li>lomiri.lomiri-app-launch</li>
    <li>lomiri.lomiri-app-launch.dev</li>
    <li>lomiri.lomiri-app-launch.doc</li>
    <li>lomiri.lomiri-calculator-app</li>
    <li>lomiri.lomiri-calendar-app</li>
    <li>lomiri.lomiri-camera-app</li>
    <li>lomiri.lomiri-clock-app</li>
    <li>lomiri.lomiri-content-hub</li>
    <li>lomiri.lomiri-content-hub.dev</li>
    <li>lomiri.lomiri-content-hub.doc</li>
    <li>lomiri.lomiri-content-hub.examples</li>
    <li>lomiri.lomiri-docviewer-app</li>
    <li>lomiri.lomiri-filemanager-app</li>
    <li>lomiri.lomiri-gallery-app</li>
    <li>lomiri.lomiri-indicator-datetime</li>
    <li>lomiri.lomiri-indicator-network</li>
    <li>lomiri.lomiri-indicator-network.dev</li>
    <li>lomiri.lomiri-indicator-network.doc</li>
    <li>lomiri.lomiri-mediaplayer-app</li>
    <li>lomiri.lomiri-music-app</li>
    <li>lomiri.lomiri-push-qml</li>
    <li>lomiri.lomiri-session</li>
    <li>lomiri.lomiri-system-settings</li>
    <li>lomiri.lomiri-system-settings-unwrapped</li>
    <li>lomiri.lomiri-system-settings-unwrapped.dev</li>
    <li>lomiri.lomiri-telephony-service</li>
    <li>lomiri.lomiri-terminal-app</li>
    <li>lomiri.lomiri-ui-extras</li>
    <li>lomiri.lomiri-ui-toolkit</li>
    <li>lomiri.lomiri-ui-toolkit.dev</li>
    <li>lomiri.lomiri-ui-toolkit.doc</li>
    <li>lomiri.lomiri-url-dispatcher</li>
    <li>lomiri.lomiri-url-dispatcher.dev</li>
    <li>lomiri.lomiri-url-dispatcher.lib</li>
    <li>lomiri.qtmir</li>
    <li>lomiri.qtmir.dev</li>
    <li>lomiri.teleports</li>
    <li>lunacy</li>
    <li>m32edit</li>
    <li>mailspring</li>
    <li>marvin</li>
    <li>mfc465cncupswrapper</li>
    <li>mfc465cnlpr</li>
    <li>mfc5890cncupswrapper</li>
    <li>mfc5890cnlpr</li>
    <li>mfc9140cdncupswrapper</li>
    <li>mfc9140cdnlpr</li>
    <li>mfcj470dw-cupswrapper</li>
    <li>mfcj470dwlpr</li>
    <li>mfcj6510dw-cupswrapper</li>
    <li>mfcj6510dwlpr</li>
    <li>mfcj880dwcupswrapper</li>
    <li>mfcj880dwlpr</li>
    <li>mfcl2700dncupswrapper</li>
    <li>mfcl2720dwcupswrapper</li>
    <li>mfcl2720dwlpr</li>
    <li>mfcl2740dwcupswrapper</li>
    <li>mfcl2740dwlpr</li>
    <li>mfcl3770cdwcupswrapper</li>
    <li>mfcl3770cdwlpr</li>
    <li>mfcl5750dw</li>
    <li>mfcl8690cdwcupswrapper</li>
    <li>mfcl8690cdwlpr</li>
    <li>microsoft-edge</li>
    <li>microsoft-identity-broker</li>
    <li>molly-guard</li>
    <li>mongodb-compass</li>
    <li>morgen</li>
    <li>msalsdk-dbusclient</li>
    <li>multiviewer-for-f1</li>
    <li>museeks</li>
    <li>networkaudiod</li>
    <li>nxwitness-client</li>
    <li>ocenaudio</li>
    <li>odoo</li>
    <li>odoo.dist</li>
    <li>onlyoffice-desktopeditors</li>
    <li>onlyoffice-documentserver</li>
    <li>pantum-driver</li>
    <li>parsec-bin</li>
    <li>pcsc-safenet</li>
    <li>pdfstudio2021</li>
    <li>pdfstudio2022</li>
    <li>pdfstudio2023</li>
    <li>pdfstudio2024</li>
    <li>pdfstudioviewer</li>
    <li>pdftowrite</li>
    <li>pdftowrite.dist</li>
    <li>perl538Packages.pacup</li>
    <li>perlPackages.pacup (perl540Packages.pacup)</li>
    <li>picoscope</li>
    <li>pike</li>
    <li>pixeluvo</li>
    <li>plex</li>
    <li>plexRaw</li>
    <li>plexRaw.basedb</li>
    <li>polar-bookshelf</li>
    <li>polar-bookshelf1</li>
    <li>positron-bin</li>
    <li>proton-authenticator</li>
    <li>proton-pass</li>
    <li>protonmail-desktop</li>
    <li>protonvpn-gui</li>
    <li>protonvpn-gui.dist</li>
    <li>python312Packages.proton-vpn-network-manager</li>
    <li>python312Packages.proton-vpn-network-manager.dist</li>
    <li>python312Packages.python-apt</li>
    <li>python312Packages.python-apt.dist</li>
    <li>python313Packages.proton-vpn-network-manager</li>
    <li>python313Packages.proton-vpn-network-manager.dist</li>
    <li>python313Packages.python-apt</li>
    <li>python313Packages.python-apt.dist</li>
    <li>qq</li>
    <li>qqmusic</li>
    <li>reqable</li>
    <li>rescuetime</li>
    <li>roam-research</li>
    <li>rocmPackages.hsa-amd-aqlprofile-bin (rocmPackages_6.hsa-amd-aqlprofile-bin)</li>
    <li>slack</li>
    <li>spacedrive</li>
    <li>sparkle</li>
    <li>speedify</li>
    <li>spotube</li>
    <li>sslmate-agent</li>
    <li>standardnotes</li>
    <li>staruml</li>
    <li>synology-drive-client</li>
    <li>tana</li>
    <li>terra-station</li>
    <li>tests.auto-patchelf-hook.withStructuredAttrs</li>
    <li>tests.auto-patchelf-hook.withoutStructuredAttrs</li>
    <li>tests.hardeningFlags-clang.allExplicitDisabledBindNow</li>
    <li>tests.hardeningFlags-clang.allExplicitDisabledFortify</li>
    <li>tests.hardeningFlags-clang.allExplicitDisabledStackClashProtection</li>
    <li>tests.hardeningFlags-clang.allExplicitDisabledStackProtector</li>
    <li>tests.hardeningFlags-clang.bindNowExplicitDisabled</li>
    <li>tests.hardeningFlags-clang.bindNowExplicitEnabled</li>
    <li>tests.hardeningFlags-clang.fortify1ExplicitDisabledCmdlineEnabled</li>
    <li>tests.hardeningFlags-clang.fortify1ExplicitEnabledCmdlineDisabled</li>
    <li>tests.hardeningFlags-clang.fortify3EnabledEnvEnablesFortify1</li>
    <li>tests.hardeningFlags-clang.fortify3ExplicitDisabled</li>
    <li>tests.hardeningFlags-clang.fortify3ExplicitDisabledDoesntDisableFortify</li>
    <li>tests.hardeningFlags-clang.fortify3StdenvUnsupp</li>
    <li>tests.hardeningFlags-clang.fortify3StdenvUnsuppDoesntUnsuppFortify1</li>
    <li>tests.hardeningFlags-clang.fortifyEnabledEnvDoesntEnableFortify3</li>
    <li>tests.hardeningFlags-clang.fortifyExplicitDisabled</li>
    <li>tests.hardeningFlags-clang.fortifyExplicitDisabledDisablesFortify3</li>
    <li>tests.hardeningFlags-clang.fortifyExplicitEnabled</li>
    <li>tests.hardeningFlags-clang.fortifyStdenvUnsupp</li>
    <li>tests.hardeningFlags-clang.fortifyStdenvUnsuppUnsupportsFortify3</li>
    <li>tests.hardeningFlags-clang.pieExplicitEnabled</li>
    <li>tests.hardeningFlags-clang.pieExplicitEnabledStructuredAttrs</li>
    <li>tests.hardeningFlags-clang.relROExplicitEnabled</li>
    <li>tests.hardeningFlags-clang.sfa1EnabledEnvDoesntEnableSfa3</li>
    <li>tests.hardeningFlags-clang.sfa1StdenvUnsupp</li>
    <li>tests.hardeningFlags-clang.sfa1StdenvUnsuppUnsupportsSfa3</li>
    <li>tests.hardeningFlags-clang.sfa1explicitDisabled</li>
    <li>tests.hardeningFlags-clang.sfa1explicitDisabledDisablesSfa3</li>
    <li>tests.hardeningFlags-clang.sfa1explicitEnabled</li>
    <li>tests.hardeningFlags-clang.sfa1explicitEnabledDoesntProtectDefLen1</li>
    <li>tests.hardeningFlags-clang.sfa3EnabledEnvEnablesSfa1</li>
    <li>tests.hardeningFlags-clang.sfa3StdenvUnsupp</li>
    <li>tests.hardeningFlags-clang.sfa3StdenvUnsuppDoesntUnsuppSfa1</li>
    <li>tests.hardeningFlags-clang.sfa3explicitDisabledDoesntDisableSfa1</li>
    <li>tests.hardeningFlags-clang.sfa3explicitEnabledDoesntProtectCorrectFlex</li>
    <li>tests.hardeningFlags-clang.sfa3explicitEnabledProtectsDefLen1</li>
    <li>tests.hardeningFlags-clang.stackClashProtectionExplicitDisabled</li>
    <li>tests.hardeningFlags-clang.stackClashProtectionStdenvUnsupp</li>
    <li>tests.hardeningFlags-clang.stackProtectorExplicitDisabled</li>
    <li>tests.hardeningFlags-clang.stackProtectorExplicitEnabled</li>
    <li>tests.hardeningFlags-clang.stackProtectorRedisabledEnv</li>
    <li>tests.hardeningFlags-clang.stackProtectorReenabledEnv</li>
    <li>tests.hardeningFlags-clang.stackProtectorReenabledFromAllEnv</li>
    <li>tests.hardeningFlags-clang.stackProtectorStdenvUnsupp</li>
    <li>tests.hardeningFlags-clang.stackProtectorUnsupportedEnabledEnv</li>
    <li>tests.hardeningFlags.allExplicitDisabledBindNow (tests.hardeningFlags-gcc.allExplicitDisabledBindNow)</li>
    <li>tests.hardeningFlags.allExplicitDisabledFortify (tests.hardeningFlags-gcc.allExplicitDisabledFortify)</li>
    <li>tests.hardeningFlags.allExplicitDisabledPie (tests.hardeningFlags-gcc.allExplicitDisabledPie)</li>
    <li>tests.hardeningFlags.allExplicitDisabledStackClashProtection (tests.hardeningFlags-gcc.allExplicitDisabledStackClashProtection)</li>
    <li>tests.hardeningFlags.allExplicitDisabledStackProtector (tests.hardeningFlags-gcc.allExplicitDisabledStackProtector)</li>
    <li>tests.hardeningFlags.bindNowExplicitDisabled (tests.hardeningFlags-gcc.bindNowExplicitDisabled)</li>
    <li>tests.hardeningFlags.bindNowExplicitEnabled (tests.hardeningFlags-gcc.bindNowExplicitEnabled)</li>
    <li>tests.hardeningFlags.fortify1ExplicitDisabledCmdlineEnabled (tests.hardeningFlags-gcc.fortify1ExplicitDisabledCmdlineEnabled)</li>
    <li>tests.hardeningFlags.fortify1ExplicitEnabledCmdlineDisabled (tests.hardeningFlags-gcc.fortify1ExplicitEnabledCmdlineDisabled)</li>
    <li>tests.hardeningFlags.fortify3EnabledEnvEnablesFortify1 (tests.hardeningFlags-gcc.fortify3EnabledEnvEnablesFortify1)</li>
    <li>tests.hardeningFlags.fortify3ExplicitDisabled (tests.hardeningFlags-gcc.fortify3ExplicitDisabled)</li>
    <li>tests.hardeningFlags.fortify3ExplicitDisabledDoesntDisableFortify (tests.hardeningFlags-gcc.fortify3ExplicitDisabledDoesntDisableFortify)</li>
    <li>tests.hardeningFlags.fortify3ExplicitEnabled (tests.hardeningFlags-gcc.fortify3ExplicitEnabled)</li>
    <li>tests.hardeningFlags.fortify3StdenvUnsupp (tests.hardeningFlags-gcc.fortify3StdenvUnsupp)</li>
    <li>tests.hardeningFlags.fortify3StdenvUnsuppDoesntUnsuppFortify1 (tests.hardeningFlags-gcc.fortify3StdenvUnsuppDoesntUnsuppFortify1)</li>
    <li>tests.hardeningFlags.fortifyEnabledEnvDoesntEnableFortify3 (tests.hardeningFlags-gcc.fortifyEnabledEnvDoesntEnableFortify3)</li>
    <li>tests.hardeningFlags.fortifyExplicitDisabled (tests.hardeningFlags-gcc.fortifyExplicitDisabled)</li>
    <li>tests.hardeningFlags.fortifyExplicitDisabledDisablesFortify3 (tests.hardeningFlags-gcc.fortifyExplicitDisabledDisablesFortify3)</li>
    <li>tests.hardeningFlags.fortifyExplicitEnabled (tests.hardeningFlags-gcc.fortifyExplicitEnabled)</li>
    <li>tests.hardeningFlags.fortifyStdenvUnsupp (tests.hardeningFlags-gcc.fortifyStdenvUnsupp)</li>
    <li>tests.hardeningFlags.fortifyStdenvUnsuppUnsupportsFortify3 (tests.hardeningFlags-gcc.fortifyStdenvUnsuppUnsupportsFortify3)</li>
    <li>tests.hardeningFlags.pieExplicitDisabled (tests.hardeningFlags-gcc.pieExplicitDisabled)</li>
    <li>tests.hardeningFlags.pieExplicitEnabled (tests.hardeningFlags-gcc.pieExplicitEnabled)</li>
    <li>tests.hardeningFlags.pieExplicitEnabledStructuredAttrs (tests.hardeningFlags-gcc.pieExplicitEnabledStructuredAttrs)</li>
    <li>tests.hardeningFlags.relROExplicitEnabled (tests.hardeningFlags-gcc.relROExplicitEnabled)</li>
    <li>tests.hardeningFlags.sfa1EnabledEnvDoesntEnableSfa3 (tests.hardeningFlags-gcc.sfa1EnabledEnvDoesntEnableSfa3)</li>
    <li>tests.hardeningFlags.sfa1StdenvUnsupp (tests.hardeningFlags-gcc.sfa1StdenvUnsupp)</li>
    <li>tests.hardeningFlags.sfa1StdenvUnsuppUnsupportsSfa3 (tests.hardeningFlags-gcc.sfa1StdenvUnsuppUnsupportsSfa3)</li>
    <li>tests.hardeningFlags.sfa1explicitDisabled (tests.hardeningFlags-gcc.sfa1explicitDisabled)</li>
    <li>tests.hardeningFlags.sfa1explicitDisabledDisablesSfa3 (tests.hardeningFlags-gcc.sfa1explicitDisabledDisablesSfa3)</li>
    <li>tests.hardeningFlags.sfa1explicitEnabled (tests.hardeningFlags-gcc.sfa1explicitEnabled)</li>
    <li>tests.hardeningFlags.sfa1explicitEnabledDoesntProtectDefLen1 (tests.hardeningFlags-gcc.sfa1explicitEnabledDoesntProtectDefLen1)</li>
    <li>tests.hardeningFlags.sfa3EnabledEnvEnablesSfa1 (tests.hardeningFlags-gcc.sfa3EnabledEnvEnablesSfa1)</li>
    <li>tests.hardeningFlags.sfa3StdenvUnsupp (tests.hardeningFlags-gcc.sfa3StdenvUnsupp)</li>
    <li>tests.hardeningFlags.sfa3StdenvUnsuppDoesntUnsuppSfa1 (tests.hardeningFlags-gcc.sfa3StdenvUnsuppDoesntUnsuppSfa1)</li>
    <li>tests.hardeningFlags.sfa3explicitDisabledDoesntDisableSfa1 (tests.hardeningFlags-gcc.sfa3explicitDisabledDoesntDisableSfa1)</li>
    <li>tests.hardeningFlags.sfa3explicitEnabledDoesntProtectCorrectFlex (tests.hardeningFlags-gcc.sfa3explicitEnabledDoesntProtectCorrectFlex)</li>
    <li>tests.hardeningFlags.sfa3explicitEnabledProtectsDefLen1 (tests.hardeningFlags-gcc.sfa3explicitEnabledProtectsDefLen1)</li>
    <li>tests.hardeningFlags.stackClashProtectionExplicitDisabled (tests.hardeningFlags-gcc.stackClashProtectionExplicitDisabled)</li>
    <li>tests.hardeningFlags.stackClashProtectionExplicitEnabled (tests.hardeningFlags-gcc.stackClashProtectionExplicitEnabled)</li>
    <li>tests.hardeningFlags.stackClashProtectionStdenvUnsupp (tests.hardeningFlags-gcc.stackClashProtectionStdenvUnsupp)</li>
    <li>tests.hardeningFlags.stackProtectorExplicitDisabled (tests.hardeningFlags-gcc.stackProtectorExplicitDisabled)</li>
    <li>tests.hardeningFlags.stackProtectorExplicitEnabled (tests.hardeningFlags-gcc.stackProtectorExplicitEnabled)</li>
    <li>tests.hardeningFlags.stackProtectorRedisabledEnv (tests.hardeningFlags-gcc.stackProtectorRedisabledEnv)</li>
    <li>tests.hardeningFlags.stackProtectorReenabledEnv (tests.hardeningFlags-gcc.stackProtectorReenabledEnv)</li>
    <li>tests.hardeningFlags.stackProtectorReenabledFromAllEnv (tests.hardeningFlags-gcc.stackProtectorReenabledFromAllEnv)</li>
    <li>tests.hardeningFlags.stackProtectorStdenvUnsupp (tests.hardeningFlags-gcc.stackProtectorStdenvUnsupp)</li>
    <li>tests.hardeningFlags.stackProtectorUnsupportedEnabledEnv (tests.hardeningFlags-gcc.stackProtectorUnsupportedEnabledEnv)</li>
    <li>texturepacker</li>
    <li>thedesk</li>
    <li>ticktick</li>
    <li>todesk</li>
    <li>tonelib-gfx</li>
    <li>tonelib-jam</li>
    <li>tonelib-metal</li>
    <li>tonelib-noisereducer</li>
    <li>touchosc</li>
    <li>trillian-im</li>
    <li>twingate</li>
    <li>typora</li>
    <li>ubports-click</li>
    <li>unifi</li>
    <li>unigine-superposition</li>
    <li>unityhub</li>
    <li>unixODBCDrivers.msodbcsql17</li>
    <li>unixODBCDrivers.msodbcsql18</li>
    <li>unixODBCDrivers.redshift</li>
    <li>uw-ttyp0</li>
    <li>viber</li>
    <li>waveterm</li>
    <li>webull-desktop</li>
    <li>wemeet</li>
    <li>wkhtmltopdf</li>
    <li>x32edit</li>
    <li>xmind</li>
    <li>zepp-simulator</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-linux`</summary>
<details>
<summary>jigasi</summary>
<pre>Running phase: updateAutotoolsGnuConfigScriptsPhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;updateAutotoolsGnuConfigScriptsPhase&quot; }
Running phase: configurePhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;configurePhase&quot; }
no configure script, doing nothing
Running phase: installPhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;installPhase&quot; }
substituteStream() in derivation jigasi-1.1-311-g3de47d0: WARNING: &#x27;--replace&#x27; is deprecated, use --replace-{fail,warn,quiet}. (file &#x27;usr/share/jigasi/jigasi.sh&#x27;)
Running phase: fixupPhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;fixupPhase&quot; }
shrinking RPATHs of ELF executables and libraries in /nix/store/grj8623kys23dz3v5bgkf436yg8l9bry-jigasi-1.1-311-g3de47d0
checking for references to /build/ in /nix/store/grj8623kys23dz3v5bgkf436yg8l9bry-jigasi-1.1-311-g3de47d0...
patching script interpreter paths in /nix/store/grj8623kys23dz3v5bgkf436yg8l9bry-jigasi-1.1-311-g3de47d0
/nix/store/grj8623kys23dz3v5bgkf436yg8l9bry-jigasi-1.1-311-g3de47d0/share/jigasi/collect-dump-logs.sh: interpreter directive changed from &quot;#!/bin/bash&quot; to &quot;/nix/store/q7sqwn7i6w2b67adw0bmix29pxg85x3w-bash-5.3p3/bin/bash&quot;
/nix/store/grj8623kys23dz3v5bgkf436yg8l9bry-jigasi-1.1-311-g3de47d0/share/jigasi/graceful_shutdown.sh: interpreter directive changed from &quot;#!/bin/bash&quot; to &quot;/nix/store/q7sqwn7i6w2b67adw0bmix29pxg85x3w-bash-5.3p3/bin/bash&quot;
/nix/store/grj8623kys23dz3v5bgkf436yg8l9bry-jigasi-1.1-311-g3de47d0/share/jigasi/jigasi.sh: interpreter directive changed from &quot;#!/bin/bash&quot; to &quot;/nix/store/q7sqwn7i6w2b67adw0bmix29pxg85x3w-bash-5.3p3/bin/bash&quot;
/nix/store/grj8623kys23dz3v5bgkf436yg8l9bry-jigasi-1.1-311-g3de47d0/etc/init.d/jigasi: interpreter directive changed from &quot;#! /bin/sh&quot; to &quot;/nix/store/q7sqwn7i6w2b67adw0bmix29pxg85x3w-bash-5.3p3/bin/sh&quot;
stripping (with command strip and flags -S -p) in  /nix/store/grj8623kys23dz3v5bgkf436yg8l9bry-jigasi-1.1-311-g3de47d0/bin
ERROR: noBrokenSymlinks: the symlink /nix/store/grj8623kys23dz3v5bgkf436yg8l9bry-jigasi-1.1-311-g3de47d0/share/jigasi/lib/libunix-java.so points to a missing target: /nix/store/grj8623kys23dz3v5bgkf436yg8l9bry-jigasi-1.1-311-g3de47d0/share/jigasi/lib/libunix-0.5.1.so
ERROR: noBrokenSymlinks: found 1 dangling symlinks, 0 reflexive symlinks and 0 unreadable symlinks</pre>
</details>
<details>
<summary>mullvad-vpn</summary>
<pre>    libdbus-1.so.3 -&gt; found: /nix/store/wmaqf8a3d2hzljx92sckpa4ivzqh5dcm-dbus-1.14.10-lib/lib
    libgcc_s.so.1 -&gt; found: /nix/store/8zm936ys1dkqv2710j0w09r2mk0kj5n9-gcc-14.3.0-libgcc/lib
setting RPATH to: /nix/store/acjdidq41qig9khxcm7gx1d7brzjs249-systemd-257.8/lib:/nix/store/pgxicv9xgn7y4wlamnlwzh2frlcfi2f5-libglvnd-1.7.0/lib:/nix/store/g7q1nwk40gby2x5k4811r1m66d681pca-libnotify-0.8.6/lib:/nix/store/g1w363lk3x35fr1m12nkraxk1fsp4vrm-libappindicator-gtk3-12.10.1+20.10.20200706.1/lib:/nix/store/yb3bf4iyxzjhs51kgj3w7kw42ci9vx3n-wayland-1.24.0/lib:/nix/store/wmaqf8a3d2hzljx92sckpa4ivzqh5dcm-dbus-1.14.10-lib/lib:/nix/store/8zm936ys1dkqv2710j0w09r2mk0kj5n9-gcc-14.3.0-libgcc/lib
setting interpreter of /nix/store/13aygh7rvglnklw7fvccj5p73q6w1xi5-mullvad-vpn-2025.7/share/mullvad/resources/mullvad-setup
searching for dependencies of /nix/store/13aygh7rvglnklw7fvccj5p73q6w1xi5-mullvad-vpn-2025.7/share/mullvad/resources/mullvad-setup
    libgcc_s.so.1 -&gt; found: /nix/store/8zm936ys1dkqv2710j0w09r2mk0kj5n9-gcc-14.3.0-libgcc/lib
setting RPATH to: /nix/store/acjdidq41qig9khxcm7gx1d7brzjs249-systemd-257.8/lib:/nix/store/pgxicv9xgn7y4wlamnlwzh2frlcfi2f5-libglvnd-1.7.0/lib:/nix/store/g7q1nwk40gby2x5k4811r1m66d681pca-libnotify-0.8.6/lib:/nix/store/g1w363lk3x35fr1m12nkraxk1fsp4vrm-libappindicator-gtk3-12.10.1+20.10.20200706.1/lib:/nix/store/yb3bf4iyxzjhs51kgj3w7kw42ci9vx3n-wayland-1.24.0/lib:/nix/store/8zm936ys1dkqv2710j0w09r2mk0kj5n9-gcc-14.3.0-libgcc/lib
setting interpreter of /nix/store/13aygh7rvglnklw7fvccj5p73q6w1xi5-mullvad-vpn-2025.7/share/mullvad/resources/openvpn
searching for dependencies of /nix/store/13aygh7rvglnklw7fvccj5p73q6w1xi5-mullvad-vpn-2025.7/share/mullvad/resources/openvpn
setting RPATH to: /nix/store/acjdidq41qig9khxcm7gx1d7brzjs249-systemd-257.8/lib:/nix/store/pgxicv9xgn7y4wlamnlwzh2frlcfi2f5-libglvnd-1.7.0/lib:/nix/store/g7q1nwk40gby2x5k4811r1m66d681pca-libnotify-0.8.6/lib:/nix/store/g1w363lk3x35fr1m12nkraxk1fsp4vrm-libappindicator-gtk3-12.10.1+20.10.20200706.1/lib:/nix/store/yb3bf4iyxzjhs51kgj3w7kw42ci9vx3n-wayland-1.24.0/lib
auto-patchelf: 0 dependencies could not be satisfied
rewriting symlink /nix/store/13aygh7rvglnklw7fvccj5p73q6w1xi5-mullvad-vpn-2025.7/bin/mullvad-problem-report to be relative to /nix/store/13aygh7rvglnklw7fvccj5p73q6w1xi5-mullvad-vpn-2025.7
rewriting symlink /nix/store/13aygh7rvglnklw7fvccj5p73q6w1xi5-mullvad-vpn-2025.7/bin/.mullvad-vpn-wrapped to be relative to /nix/store/13aygh7rvglnklw7fvccj5p73q6w1xi5-mullvad-vpn-2025.7
rewriting symlink /nix/store/13aygh7rvglnklw7fvccj5p73q6w1xi5-mullvad-vpn-2025.7/bin/.mullvad-gui-wrapped to be relative to /nix/store/13aygh7rvglnklw7fvccj5p73q6w1xi5-mullvad-vpn-2025.7
Running phase: installCheckPhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;installCheckPhase&quot; }
Executing versionCheckPhase
versionCheckHook: Package `mullvad-vpn` does not have the `meta.mainProgram` attribute. We&#x27;ll assume that the main program has the same name for now, but this behavior is deprecated, because it leads to surprising errors when the assumption does not hold. If the package has a main program, please set `meta.mainProgram` in its definition to make this warning go away. Should the binary that outputs the intended version differ from `meta.mainProgram`, consider setting `versionCheckProgram` instead.
Did not find version 2025.7 in the output of the command /nix/store/13aygh7rvglnklw7fvccj5p73q6w1xi5-mullvad-vpn-2025.7/bin/mullvad-vpn --version
[253:0924/185015.719651:FATAL:setuid_sandbox_host.cc(163)] The SUID sandbox helper binary was found, but is not configured correctly. Rather than run without sandboxing I&#x27;m aborting now. You need to make sure that {{storeDir}}/share/mullvad/chrome-sandbox is owned by root and has mode 4755.</pre>
</details>
<details>
<summary>tests.buildFHSEnv.liblzma</summary>
<pre>original ldd output is:
ldd: $warning: you do not have execution permission for `/nix/store/1pli0ajw766zbgqc73hj2cq9i39516md-libxml2.so.2.9.14-fetcher&#x27;
	linux-vdso.so.1 (0x00007ffff7fc4000)
	libicuuc.so.72 =&gt; not found
	libz.so.1 =&gt; not found
	liblzma.so.5 =&gt; not found
	libm.so.6 =&gt; /nix/store/776irwlgfb65a782cxmyk61pck460fs9-glibc-2.40-66/lib/libm.so.6 (0x00007ffff7d2a000)
	libc.so.6 =&gt; /nix/store/776irwlgfb65a782cxmyk61pck460fs9-glibc-2.40-66/lib/libc.so.6 (0x00007ffff7a00000)
	/nix/store/776irwlgfb65a782cxmyk61pck460fs9-glibc-2.40-66/lib64/ld-linux-x86-64.so.2 (0x00007ffff7fc6000)
bwrap: setting up uid map: Permission denied</pre>
</details>
<details>
<summary>tests.hardeningFlags-clang.allExplicitDisabledPie</summary>
<pre>/nix/store/y1dlhiab9hp8031358p7wq83js8sxdmx-test-bin/bin/test-bin:
 Position Independent Executable: yes
 Stack protected: no, not found! (ignored)
 Fortify Source functions: no, only unprotected functions found! (ignored)
 Read-only relocations: yes
 Immediate binding: no, not found! (ignored)
 Stack clash protection: unknown, no -fstack-clash-protection instructions found
 Control flow integrity: no, not found! (ignored)
 Branch Protection: no, not found! (ignored)
ERROR: Expected hardening-check to fail, but it passed!</pre>
</details>
<details>
<summary>tests.hardeningFlags-clang.pieExplicitDisabled</summary>
<pre>/nix/store/amnw185bwa2ikj3y6xzqgqdyda4xmmdc-test-bin/bin/test-bin:
 Position Independent Executable: yes
 Stack protected: yes
 Fortify Source functions: yes
 Read-only relocations: yes
 Immediate binding: yes
 Stack clash protection: unknown, no -fstack-clash-protection instructions found
 Control flow integrity: no, not found! (ignored)
 Branch Protection: no, not found! (ignored)
ERROR: Expected hardening-check to fail, but it passed!</pre>
</details>
</details>


---
### `aarch64-linux`
<details>
  <summary>:fast_forward: 6 packages marked as broken and skipped:</summary>
  <ul>
    <li>beekeeper-studio</li>
    <li>bitcomet</li>
    <li>deltatouch</li>
    <li>holochain-launcher</li>
    <li>lomiri.morph-browser</li>
    <li>lomiri.morph-browser.doc</li>
  </ul>
</details>
<details>
  <summary>:x: 13 packages failed to build:</summary>
  <ul>
    <li>hqplayer-desktop</li>
    <li>hqplayer-desktop.doc</li>
    <li>jigasi</li>
    <li>mullvad-vpn</li>
    <li>odoo16</li>
    <li>odoo16.dist</li>
    <li>odoo17</li>
    <li>odoo17.dist</li>
    <li>pike</li>
    <li>tests.hardeningFlags-clang.allExplicitDisabledPie</li>
    <li>tests.hardeningFlags-clang.pieExplicitDisabled</li>
    <li>wechat-uos</li>
    <li>xunlei-uos</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 226 packages built:</summary>
  <ul>
    <li>airtame</li>
    <li>apkg</li>
    <li>apkg.dist</li>
    <li>appgate-sdp</li>
    <li>apt</li>
    <li>apt.dev</li>
    <li>apt.doc</li>
    <li>apt.man</li>
    <li>ayatana-indicator-bluetooth</li>
    <li>ayatana-indicator-datetime</li>
    <li>ayatana-indicator-display</li>
    <li>ayatana-indicator-power</li>
    <li>ayatana-indicator-session</li>
    <li>ayatana-indicator-sound</li>
    <li>bdf2psf</li>
    <li>bilibili</li>
    <li>bisq2</li>
    <li>blackfire</li>
    <li>brave</li>
    <li>cargo-deb</li>
    <li>cloudflare-warp</li>
    <li>cozette</li>
    <li>dcp375cw-cupswrapper</li>
    <li>dcp9020cdw-cupswrapper</li>
    <li>debian-devscripts</li>
    <li>debootstrap</li>
    <li>debos</li>
    <li>distrobuilder</li>
    <li>dpkg</li>
    <li>feishu</li>
    <li>fflinuxprint</li>
    <li>git-buildpackage</li>
    <li>git-buildpackage.dist</li>
    <li>gohufont</li>
    <li>greenfoot</li>
    <li>guymager</li>
    <li>hydra</li>
    <li>hydra.doc</li>
    <li>jibri</li>
    <li>jicofo</li>
    <li>jitsi-meet-prosody</li>
    <li>jitsi-videobridge</li>
    <li>kodelife</li>
    <li>koreader</li>
    <li>libayatana-common</li>
    <li>lomiri.libusermetrics</li>
    <li>lomiri.libusermetrics.dev</li>
    <li>lomiri.libusermetrics.doc</li>
    <li>lomiri.lomiri</li>
    <li>lomiri.lomiri-app-launch</li>
    <li>lomiri.lomiri-app-launch.dev</li>
    <li>lomiri.lomiri-app-launch.doc</li>
    <li>lomiri.lomiri-calculator-app</li>
    <li>lomiri.lomiri-calendar-app</li>
    <li>lomiri.lomiri-camera-app</li>
    <li>lomiri.lomiri-clock-app</li>
    <li>lomiri.lomiri-content-hub</li>
    <li>lomiri.lomiri-content-hub.dev</li>
    <li>lomiri.lomiri-content-hub.doc</li>
    <li>lomiri.lomiri-content-hub.examples</li>
    <li>lomiri.lomiri-docviewer-app</li>
    <li>lomiri.lomiri-filemanager-app</li>
    <li>lomiri.lomiri-gallery-app</li>
    <li>lomiri.lomiri-indicator-datetime</li>
    <li>lomiri.lomiri-indicator-network</li>
    <li>lomiri.lomiri-indicator-network.dev</li>
    <li>lomiri.lomiri-indicator-network.doc</li>
    <li>lomiri.lomiri-mediaplayer-app</li>
    <li>lomiri.lomiri-music-app</li>
    <li>lomiri.lomiri-push-qml</li>
    <li>lomiri.lomiri-session</li>
    <li>lomiri.lomiri-system-settings</li>
    <li>lomiri.lomiri-system-settings-unwrapped</li>
    <li>lomiri.lomiri-system-settings-unwrapped.dev</li>
    <li>lomiri.lomiri-telephony-service</li>
    <li>lomiri.lomiri-terminal-app</li>
    <li>lomiri.lomiri-ui-extras</li>
    <li>lomiri.lomiri-ui-toolkit</li>
    <li>lomiri.lomiri-ui-toolkit.dev</li>
    <li>lomiri.lomiri-ui-toolkit.doc</li>
    <li>lomiri.lomiri-url-dispatcher</li>
    <li>lomiri.lomiri-url-dispatcher.dev</li>
    <li>lomiri.lomiri-url-dispatcher.lib</li>
    <li>lomiri.qtmir</li>
    <li>lomiri.qtmir.dev</li>
    <li>lomiri.teleports</li>
    <li>lunacy</li>
    <li>m32edit</li>
    <li>marvin</li>
    <li>mlxbf-bootimages</li>
    <li>molly-guard</li>
    <li>networkaudiod</li>
    <li>odoo</li>
    <li>odoo.dist</li>
    <li>onlyoffice-documentserver</li>
    <li>parsec-bin</li>
    <li>pdfstudio2021</li>
    <li>pdfstudio2022</li>
    <li>pdfstudio2023</li>
    <li>pdfstudio2024</li>
    <li>pdfstudioviewer</li>
    <li>pdftowrite</li>
    <li>pdftowrite.dist</li>
    <li>perl538Packages.pacup</li>
    <li>perlPackages.pacup (perl540Packages.pacup)</li>
    <li>plex</li>
    <li>plexRaw</li>
    <li>plexRaw.basedb</li>
    <li>polar-bookshelf</li>
    <li>polar-bookshelf1</li>
    <li>positron-bin</li>
    <li>protonvpn-gui</li>
    <li>protonvpn-gui.dist</li>
    <li>python312Packages.proton-vpn-network-manager</li>
    <li>python312Packages.proton-vpn-network-manager.dist</li>
    <li>python312Packages.python-apt</li>
    <li>python312Packages.python-apt.dist</li>
    <li>python313Packages.proton-vpn-network-manager</li>
    <li>python313Packages.proton-vpn-network-manager.dist</li>
    <li>python313Packages.python-apt</li>
    <li>python313Packages.python-apt.dist</li>
    <li>qq</li>
    <li>sparkle</li>
    <li>spotube</li>
    <li>sslmate-agent</li>
    <li>standardnotes</li>
    <li>tana</li>
    <li>tests.hardeningFlags-clang.allExplicitDisabledBindNow</li>
    <li>tests.hardeningFlags-clang.allExplicitDisabledFortify</li>
    <li>tests.hardeningFlags-clang.allExplicitDisabledStackProtector</li>
    <li>tests.hardeningFlags-clang.bindNowExplicitDisabled</li>
    <li>tests.hardeningFlags-clang.bindNowExplicitEnabled</li>
    <li>tests.hardeningFlags-clang.fortify1ExplicitDisabledCmdlineEnabled</li>
    <li>tests.hardeningFlags-clang.fortify1ExplicitEnabledCmdlineDisabled</li>
    <li>tests.hardeningFlags-clang.fortify3EnabledEnvEnablesFortify1</li>
    <li>tests.hardeningFlags-clang.fortify3ExplicitDisabled</li>
    <li>tests.hardeningFlags-clang.fortify3ExplicitDisabledDoesntDisableFortify</li>
    <li>tests.hardeningFlags-clang.fortify3StdenvUnsupp</li>
    <li>tests.hardeningFlags-clang.fortify3StdenvUnsuppDoesntUnsuppFortify1</li>
    <li>tests.hardeningFlags-clang.fortifyEnabledEnvDoesntEnableFortify3</li>
    <li>tests.hardeningFlags-clang.fortifyExplicitDisabled</li>
    <li>tests.hardeningFlags-clang.fortifyExplicitDisabledDisablesFortify3</li>
    <li>tests.hardeningFlags-clang.fortifyExplicitEnabled</li>
    <li>tests.hardeningFlags-clang.fortifyStdenvUnsupp</li>
    <li>tests.hardeningFlags-clang.fortifyStdenvUnsuppUnsupportsFortify3</li>
    <li>tests.hardeningFlags-clang.pieExplicitEnabled</li>
    <li>tests.hardeningFlags-clang.pieExplicitEnabledStructuredAttrs</li>
    <li>tests.hardeningFlags-clang.relROExplicitEnabled</li>
    <li>tests.hardeningFlags-clang.sfa1EnabledEnvDoesntEnableSfa3</li>
    <li>tests.hardeningFlags-clang.sfa1StdenvUnsupp</li>
    <li>tests.hardeningFlags-clang.sfa1StdenvUnsuppUnsupportsSfa3</li>
    <li>tests.hardeningFlags-clang.sfa1explicitDisabled</li>
    <li>tests.hardeningFlags-clang.sfa1explicitDisabledDisablesSfa3</li>
    <li>tests.hardeningFlags-clang.sfa1explicitEnabled</li>
    <li>tests.hardeningFlags-clang.sfa1explicitEnabledDoesntProtectDefLen1</li>
    <li>tests.hardeningFlags-clang.sfa3EnabledEnvEnablesSfa1</li>
    <li>tests.hardeningFlags-clang.sfa3StdenvUnsupp</li>
    <li>tests.hardeningFlags-clang.sfa3StdenvUnsuppDoesntUnsuppSfa1</li>
    <li>tests.hardeningFlags-clang.sfa3explicitDisabledDoesntDisableSfa1</li>
    <li>tests.hardeningFlags-clang.sfa3explicitEnabledDoesntProtectCorrectFlex</li>
    <li>tests.hardeningFlags-clang.sfa3explicitEnabledProtectsDefLen1</li>
    <li>tests.hardeningFlags-clang.stackProtectorExplicitDisabled</li>
    <li>tests.hardeningFlags-clang.stackProtectorExplicitEnabled</li>
    <li>tests.hardeningFlags-clang.stackProtectorRedisabledEnv</li>
    <li>tests.hardeningFlags-clang.stackProtectorReenabledEnv</li>
    <li>tests.hardeningFlags-clang.stackProtectorReenabledFromAllEnv</li>
    <li>tests.hardeningFlags-clang.stackProtectorStdenvUnsupp</li>
    <li>tests.hardeningFlags-clang.stackProtectorUnsupportedEnabledEnv</li>
    <li>tests.hardeningFlags.allExplicitDisabledBindNow (tests.hardeningFlags-gcc.allExplicitDisabledBindNow)</li>
    <li>tests.hardeningFlags.allExplicitDisabledFortify (tests.hardeningFlags-gcc.allExplicitDisabledFortify)</li>
    <li>tests.hardeningFlags.allExplicitDisabledPie (tests.hardeningFlags-gcc.allExplicitDisabledPie)</li>
    <li>tests.hardeningFlags.allExplicitDisabledStackProtector (tests.hardeningFlags-gcc.allExplicitDisabledStackProtector)</li>
    <li>tests.hardeningFlags.bindNowExplicitDisabled (tests.hardeningFlags-gcc.bindNowExplicitDisabled)</li>
    <li>tests.hardeningFlags.bindNowExplicitEnabled (tests.hardeningFlags-gcc.bindNowExplicitEnabled)</li>
    <li>tests.hardeningFlags.fortify1ExplicitDisabledCmdlineEnabled (tests.hardeningFlags-gcc.fortify1ExplicitDisabledCmdlineEnabled)</li>
    <li>tests.hardeningFlags.fortify1ExplicitEnabledCmdlineDisabled (tests.hardeningFlags-gcc.fortify1ExplicitEnabledCmdlineDisabled)</li>
    <li>tests.hardeningFlags.fortify3EnabledEnvEnablesFortify1 (tests.hardeningFlags-gcc.fortify3EnabledEnvEnablesFortify1)</li>
    <li>tests.hardeningFlags.fortify3ExplicitDisabled (tests.hardeningFlags-gcc.fortify3ExplicitDisabled)</li>
    <li>tests.hardeningFlags.fortify3ExplicitDisabledDoesntDisableFortify (tests.hardeningFlags-gcc.fortify3ExplicitDisabledDoesntDisableFortify)</li>
    <li>tests.hardeningFlags.fortify3ExplicitEnabled (tests.hardeningFlags-gcc.fortify3ExplicitEnabled)</li>
    <li>tests.hardeningFlags.fortify3StdenvUnsupp (tests.hardeningFlags-gcc.fortify3StdenvUnsupp)</li>
    <li>tests.hardeningFlags.fortify3StdenvUnsuppDoesntUnsuppFortify1 (tests.hardeningFlags-gcc.fortify3StdenvUnsuppDoesntUnsuppFortify1)</li>
    <li>tests.hardeningFlags.fortifyEnabledEnvDoesntEnableFortify3 (tests.hardeningFlags-gcc.fortifyEnabledEnvDoesntEnableFortify3)</li>
    <li>tests.hardeningFlags.fortifyExplicitDisabled (tests.hardeningFlags-gcc.fortifyExplicitDisabled)</li>
    <li>tests.hardeningFlags.fortifyExplicitDisabledDisablesFortify3 (tests.hardeningFlags-gcc.fortifyExplicitDisabledDisablesFortify3)</li>
    <li>tests.hardeningFlags.fortifyExplicitEnabled (tests.hardeningFlags-gcc.fortifyExplicitEnabled)</li>
    <li>tests.hardeningFlags.fortifyStdenvUnsupp (tests.hardeningFlags-gcc.fortifyStdenvUnsupp)</li>
    <li>tests.hardeningFlags.fortifyStdenvUnsuppUnsupportsFortify3 (tests.hardeningFlags-gcc.fortifyStdenvUnsuppUnsupportsFortify3)</li>
    <li>tests.hardeningFlags.pieExplicitDisabled (tests.hardeningFlags-gcc.pieExplicitDisabled)</li>
    <li>tests.hardeningFlags.pieExplicitEnabled (tests.hardeningFlags-gcc.pieExplicitEnabled)</li>
    <li>tests.hardeningFlags.pieExplicitEnabledStructuredAttrs (tests.hardeningFlags-gcc.pieExplicitEnabledStructuredAttrs)</li>
    <li>tests.hardeningFlags.relROExplicitEnabled (tests.hardeningFlags-gcc.relROExplicitEnabled)</li>
    <li>tests.hardeningFlags.sfa1EnabledEnvDoesntEnableSfa3 (tests.hardeningFlags-gcc.sfa1EnabledEnvDoesntEnableSfa3)</li>
    <li>tests.hardeningFlags.sfa1StdenvUnsupp (tests.hardeningFlags-gcc.sfa1StdenvUnsupp)</li>
    <li>tests.hardeningFlags.sfa1StdenvUnsuppUnsupportsSfa3 (tests.hardeningFlags-gcc.sfa1StdenvUnsuppUnsupportsSfa3)</li>
    <li>tests.hardeningFlags.sfa1explicitDisabled (tests.hardeningFlags-gcc.sfa1explicitDisabled)</li>
    <li>tests.hardeningFlags.sfa1explicitDisabledDisablesSfa3 (tests.hardeningFlags-gcc.sfa1explicitDisabledDisablesSfa3)</li>
    <li>tests.hardeningFlags.sfa1explicitEnabled (tests.hardeningFlags-gcc.sfa1explicitEnabled)</li>
    <li>tests.hardeningFlags.sfa1explicitEnabledDoesntProtectDefLen1 (tests.hardeningFlags-gcc.sfa1explicitEnabledDoesntProtectDefLen1)</li>
    <li>tests.hardeningFlags.sfa3EnabledEnvEnablesSfa1 (tests.hardeningFlags-gcc.sfa3EnabledEnvEnablesSfa1)</li>
    <li>tests.hardeningFlags.sfa3StdenvUnsupp (tests.hardeningFlags-gcc.sfa3StdenvUnsupp)</li>
    <li>tests.hardeningFlags.sfa3StdenvUnsuppDoesntUnsuppSfa1 (tests.hardeningFlags-gcc.sfa3StdenvUnsuppDoesntUnsuppSfa1)</li>
    <li>tests.hardeningFlags.sfa3explicitDisabledDoesntDisableSfa1 (tests.hardeningFlags-gcc.sfa3explicitDisabledDoesntDisableSfa1)</li>
    <li>tests.hardeningFlags.sfa3explicitEnabledDoesntProtectCorrectFlex (tests.hardeningFlags-gcc.sfa3explicitEnabledDoesntProtectCorrectFlex)</li>
    <li>tests.hardeningFlags.sfa3explicitEnabledProtectsDefLen1 (tests.hardeningFlags-gcc.sfa3explicitEnabledProtectsDefLen1)</li>
    <li>tests.hardeningFlags.stackProtectorExplicitDisabled (tests.hardeningFlags-gcc.stackProtectorExplicitDisabled)</li>
    <li>tests.hardeningFlags.stackProtectorExplicitEnabled (tests.hardeningFlags-gcc.stackProtectorExplicitEnabled)</li>
    <li>tests.hardeningFlags.stackProtectorRedisabledEnv (tests.hardeningFlags-gcc.stackProtectorRedisabledEnv)</li>
    <li>tests.hardeningFlags.stackProtectorReenabledEnv (tests.hardeningFlags-gcc.stackProtectorReenabledEnv)</li>
    <li>tests.hardeningFlags.stackProtectorReenabledFromAllEnv (tests.hardeningFlags-gcc.stackProtectorReenabledFromAllEnv)</li>
    <li>tests.hardeningFlags.stackProtectorStdenvUnsupp (tests.hardeningFlags-gcc.stackProtectorStdenvUnsupp)</li>
    <li>tests.hardeningFlags.stackProtectorUnsupportedEnabledEnv (tests.hardeningFlags-gcc.stackProtectorUnsupportedEnabledEnv)</li>
    <li>ticktick</li>
    <li>touchosc</li>
    <li>ubports-click</li>
    <li>unifi</li>
    <li>unixODBCDrivers.msodbcsql17</li>
    <li>unixODBCDrivers.msodbcsql18</li>
    <li>unixODBCDrivers.redshift</li>
    <li>uw-ttyp0</li>
    <li>waveterm</li>
    <li>wemeet</li>
    <li>wkhtmltopdf</li>
    <li>x32edit</li>
    <li>xmind</li>
    <li>zepp-simulator</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `aarch64-linux`</summary>
<details>
<summary>jigasi</summary>
<pre>Running phase: updateAutotoolsGnuConfigScriptsPhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;updateAutotoolsGnuConfigScriptsPhase&quot; }
Running phase: configurePhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;configurePhase&quot; }
no configure script, doing nothing
Running phase: installPhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;installPhase&quot; }
substituteStream() in derivation jigasi-1.1-311-g3de47d0: WARNING: &#x27;--replace&#x27; is deprecated, use --replace-{fail,warn,quiet}. (file &#x27;usr/share/jigasi/jigasi.sh&#x27;)
Running phase: fixupPhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;fixupPhase&quot; }
shrinking RPATHs of ELF executables and libraries in /nix/store/9kh1r038jw1yk22dlc6ml2zxfd4njgqn-jigasi-1.1-311-g3de47d0
checking for references to /build/ in /nix/store/9kh1r038jw1yk22dlc6ml2zxfd4njgqn-jigasi-1.1-311-g3de47d0...
patching script interpreter paths in /nix/store/9kh1r038jw1yk22dlc6ml2zxfd4njgqn-jigasi-1.1-311-g3de47d0
/nix/store/9kh1r038jw1yk22dlc6ml2zxfd4njgqn-jigasi-1.1-311-g3de47d0/share/jigasi/collect-dump-logs.sh: interpreter directive changed from &quot;#!/bin/bash&quot; to &quot;/nix/store/hxmi7d6vbdgbzklm4icfk2y83ncw8la9-bash-5.3p3/bin/bash&quot;
/nix/store/9kh1r038jw1yk22dlc6ml2zxfd4njgqn-jigasi-1.1-311-g3de47d0/share/jigasi/graceful_shutdown.sh: interpreter directive changed from &quot;#!/bin/bash&quot; to &quot;/nix/store/hxmi7d6vbdgbzklm4icfk2y83ncw8la9-bash-5.3p3/bin/bash&quot;
/nix/store/9kh1r038jw1yk22dlc6ml2zxfd4njgqn-jigasi-1.1-311-g3de47d0/share/jigasi/jigasi.sh: interpreter directive changed from &quot;#!/bin/bash&quot; to &quot;/nix/store/hxmi7d6vbdgbzklm4icfk2y83ncw8la9-bash-5.3p3/bin/bash&quot;
/nix/store/9kh1r038jw1yk22dlc6ml2zxfd4njgqn-jigasi-1.1-311-g3de47d0/etc/init.d/jigasi: interpreter directive changed from &quot;#! /bin/sh&quot; to &quot;/nix/store/hxmi7d6vbdgbzklm4icfk2y83ncw8la9-bash-5.3p3/bin/sh&quot;
stripping (with command strip and flags -S -p) in  /nix/store/9kh1r038jw1yk22dlc6ml2zxfd4njgqn-jigasi-1.1-311-g3de47d0/bin
ERROR: noBrokenSymlinks: the symlink /nix/store/9kh1r038jw1yk22dlc6ml2zxfd4njgqn-jigasi-1.1-311-g3de47d0/share/jigasi/lib/libunix-java.so points to a missing target: /nix/store/9kh1r038jw1yk22dlc6ml2zxfd4njgqn-jigasi-1.1-311-g3de47d0/share/jigasi/lib/libunix-0.5.1.so
ERROR: noBrokenSymlinks: found 1 dangling symlinks, 0 reflexive symlinks and 0 unreadable symlinks</pre>
</details>
<details>
<summary>mullvad-vpn</summary>
<pre>    libgcc_s.so.1 -&gt; found: /nix/store/3161xhazkr55b1vi5h0a43zk377kf4sq-gcc-14.3.0-libgcc/lib
setting RPATH to: /nix/store/psk3q50fzri1v50qigq5ffbmkcw67d1h-systemd-257.8/lib:/nix/store/wk994cq6kf682saqynadrkdbi8qa8vx1-libglvnd-1.7.0/lib:/nix/store/9mpqsbc039f1a320hc98hli3zbab5bnp-libnotify-0.8.6/lib:/nix/store/psk4sl6zfqbj18cpc86mz7r0sg925vig-libappindicator-gtk3-12.10.1+20.10.20200706.1/lib:/nix/store/vk20swk54pm6ra2w35ra9n8xjhhxh9r8-wayland-1.24.0/lib:/nix/store/g1dnz3bs3cvvihsmkjx9dnbw7gai0gz4-dbus-1.14.10-lib/lib:/nix/store/3161xhazkr55b1vi5h0a43zk377kf4sq-gcc-14.3.0-libgcc/lib
setting interpreter of /nix/store/n5i82w0cchnn4dcryrx0g5gir1y5y01f-mullvad-vpn-2025.7/share/mullvad/resources/mullvad-setup
searching for dependencies of /nix/store/n5i82w0cchnn4dcryrx0g5gir1y5y01f-mullvad-vpn-2025.7/share/mullvad/resources/mullvad-setup
    libgcc_s.so.1 -&gt; found: /nix/store/3161xhazkr55b1vi5h0a43zk377kf4sq-gcc-14.3.0-libgcc/lib
setting RPATH to: /nix/store/psk3q50fzri1v50qigq5ffbmkcw67d1h-systemd-257.8/lib:/nix/store/wk994cq6kf682saqynadrkdbi8qa8vx1-libglvnd-1.7.0/lib:/nix/store/9mpqsbc039f1a320hc98hli3zbab5bnp-libnotify-0.8.6/lib:/nix/store/psk4sl6zfqbj18cpc86mz7r0sg925vig-libappindicator-gtk3-12.10.1+20.10.20200706.1/lib:/nix/store/vk20swk54pm6ra2w35ra9n8xjhhxh9r8-wayland-1.24.0/lib:/nix/store/3161xhazkr55b1vi5h0a43zk377kf4sq-gcc-14.3.0-libgcc/lib
setting interpreter of /nix/store/n5i82w0cchnn4dcryrx0g5gir1y5y01f-mullvad-vpn-2025.7/share/mullvad/resources/openvpn
searching for dependencies of /nix/store/n5i82w0cchnn4dcryrx0g5gir1y5y01f-mullvad-vpn-2025.7/share/mullvad/resources/openvpn
setting RPATH to: /nix/store/psk3q50fzri1v50qigq5ffbmkcw67d1h-systemd-257.8/lib:/nix/store/wk994cq6kf682saqynadrkdbi8qa8vx1-libglvnd-1.7.0/lib:/nix/store/9mpqsbc039f1a320hc98hli3zbab5bnp-libnotify-0.8.6/lib:/nix/store/psk4sl6zfqbj18cpc86mz7r0sg925vig-libappindicator-gtk3-12.10.1+20.10.20200706.1/lib:/nix/store/vk20swk54pm6ra2w35ra9n8xjhhxh9r8-wayland-1.24.0/lib
auto-patchelf: 0 dependencies could not be satisfied
rewriting symlink /nix/store/n5i82w0cchnn4dcryrx0g5gir1y5y01f-mullvad-vpn-2025.7/bin/mullvad-problem-report to be relative to /nix/store/n5i82w0cchnn4dcryrx0g5gir1y5y01f-mullvad-vpn-2025.7
rewriting symlink /nix/store/n5i82w0cchnn4dcryrx0g5gir1y5y01f-mullvad-vpn-2025.7/bin/.mullvad-vpn-wrapped to be relative to /nix/store/n5i82w0cchnn4dcryrx0g5gir1y5y01f-mullvad-vpn-2025.7
rewriting symlink /nix/store/n5i82w0cchnn4dcryrx0g5gir1y5y01f-mullvad-vpn-2025.7/bin/.mullvad-gui-wrapped to be relative to /nix/store/n5i82w0cchnn4dcryrx0g5gir1y5y01f-mullvad-vpn-2025.7
Running phase: installCheckPhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;installCheckPhase&quot; }
Executing versionCheckPhase
versionCheckHook: Package `mullvad-vpn` does not have the `meta.mainProgram` attribute. We&#x27;ll assume that the main program has the same name for now, but this behavior is deprecated, because it leads to surprising errors when the assumption does not hold. If the package has a main program, please set `meta.mainProgram` in its definition to make this warning go away. Should the binary that outputs the intended version differ from `meta.mainProgram`, consider setting `versionCheckProgram` instead.
Did not find version 2025.7 in the output of the command /nix/store/n5i82w0cchnn4dcryrx0g5gir1y5y01f-mullvad-vpn-2025.7/bin/mullvad-vpn --version
[253:0924/183057.125046:FATAL:setuid_sandbox_host.cc(163)] The SUID sandbox helper binary was found, but is not configured correctly. Rather than run without sandboxing I&#x27;m aborting now. You need to make sure that {{storeDir}}/share/mullvad/chrome-sandbox is owned by root and has mode 4755.
[261:0924/183057.256522:ERROR:content_main_runner_impl.cc(473)] Failed to initialize cpuinfo</pre>
</details>
<details>
<summary>pike</summary>
<pre>./precompile.sh: line 135: /build/source/src/tpike: not found
precompile: /build/source/src/test-pike /build/source/bin/make_interpret_functions.pike /build/source/src/interpret_functions.h (method=QQQ)
./precompile.sh: line 135: /build/source/src/test-pike: not found
precompile: /build/source/src/pike -DNOT_INSTALLED -DPRECOMPILED_SEARCH_MORE -m/build/source/src/master.pike  /build/source/bin/make_interpret_functions.pike /build/source/src/interpret_functions.h (method=QQQQQ)
./precompile.sh: line 135: /build/source/src/pike: not found
precompile: /build/source/src/pike -DNOT_INSTALLED -DPRECOMPILED_SEARCH_MORE -m/build/source/src/master.pike  /build/source/bin/make_interpret_functions.pike /build/source/src/interpret_functions.h (method=QQQQQQ)
./precompile.sh: line 135: /build/source/src/pike: not found
precompile: /nix/store/alqyj8g5q760fc5iwhmyfsvncl482y1b-pike-bootstrap-8.0.1956/bin/pike /build/source/bin/make_interpret_functions.pike /build/source/src/interpret_functions.h (method=QQQQQQ)
precompile: /build/source/src/pike -DNOT_INSTALLED -DPRECOMPILED_SEARCH_MORE -m/build/source/src/master.pike  /build/source/bin/make_interpret_functions.pike /build/source/src/interpret_functions.h (method=Q)
./precompile.sh: line 135: /build/source/src/pike: not found
precompile: /build/source/src/tpike -DNOT_INSTALLED -DPRECOMPILED_SEARCH_MORE -m/build/source/src/master.pike  /build/source/bin/make_interpret_functions.pike /build/source/src/interpret_functions.h (method=QQ)
./precompile.sh: line 135: /build/source/src/tpike: not found
precompile: /build/source/src/test-pike /build/source/bin/make_interpret_functions.pike /build/source/src/interpret_functions.h (method=QQQ)
./precompile.sh: line 135: /build/source/src/test-pike: not found
precompile: /build/source/src/pike -DNOT_INSTALLED -DPRECOMPILED_SEARCH_MORE -m/build/source/src/master.pike  /build/source/bin/make_interpret_functions.pike /build/source/src/interpret_functions.h (method=QQQQQ)
./precompile.sh: line 135: /build/source/src/pike: not found
Failed to find a Pike executable that could run /build/source/bin/make_interpret_functions.pike.
make[1]: *** [Makefile:1273: /build/source/src/interpret_functions_fixed.h] Error 1
make[1]: Leaving directory &#x27;/build/source/src&#x27;
make: *** [Makefile:1202: module_objects] Error 2</pre>
</details>
<details>
<summary>tests.hardeningFlags-clang.allExplicitDisabledPie</summary>
<pre>/nix/store/7bzwlhvsmylgmgpjb0xa9xks6g776m55-test-bin/bin/test-bin:
 Position Independent Executable: yes
 Stack protected: no, not found! (ignored)
 Fortify Source functions: no, only unprotected functions found! (ignored)
 Read-only relocations: yes
 Immediate binding: no, not found! (ignored)
 Stack clash protection: unknown, no -fstack-clash-protection instructions found
 Control flow integrity: no, not found! (ignored)
 Branch Protection: no, not found! (ignored)
ERROR: Expected hardening-check to fail, but it passed!</pre>
</details>
<details>
<summary>tests.hardeningFlags-clang.pieExplicitDisabled</summary>
<pre>/nix/store/kaj0qixmi9zb85km2215dagd7sh9yvq4-test-bin/bin/test-bin:
 Position Independent Executable: yes
 Stack protected: yes
 Fortify Source functions: yes
 Read-only relocations: yes
 Immediate binding: yes
 Stack clash protection: unknown, no -fstack-clash-protection instructions found
 Control flow integrity: no, not found! (ignored)
 Branch Protection: no, not found! (ignored)
ERROR: Expected hardening-check to fail, but it passed!</pre>
</details>
</details>


---
### `x86_64-darwin` (sandbox = true)
<details>
  <summary>:x: 14 packages failed to build:</summary>
  <ul>
    <li>apkg</li>
    <li>apkg.dist</li>
    <li>bdf2psf</li>
    <li>cargo-deb</li>
    <li>cozette</li>
    <li>debian-devscripts</li>
    <li>dpkg</li>
    <li>git-buildpackage</li>
    <li>git-buildpackage.dist</li>
    <li>gohufont</li>
    <li>perl538Packages.pacup</li>
    <li>perlPackages.pacup (perl540Packages.pacup)</li>
    <li>sslmate-agent</li>
    <li>uw-ttyp0</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>kochi-substitute</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-darwin`</summary>
<details>
<summary>dpkg</summary>
<pre>
ERROR: 78 tests were run,
3 failed unexpectedly.
1 test was skipped.
## -------------------------- ##
## testsuite.log was created. ##
## -------------------------- ##

Please send &#x27;src/at/testsuite.log&#x27; and all information you think might help:

   To: &lt;debian-dpkg@lists.debian.org&gt;
   Subject: [dpkg 1.22.21] testsuite: 45 51 57 failed

You may investigate any problem if you feel able to do so, in which
case the test suite provides a good starting point.  Its output may
be found below &#x27;src/at/testsuite.dir&#x27;.

make[1]: *** [Makefile:1458: autotest-installcheck] Error 1
make[1]: Leaving directory &#x27;/nix/build/nix-14688-2909043042/dpkg/src&#x27;
make: *** [Makefile:769: installcheck-recursive] Error 1</pre>
</details>
</details>


---
### `aarch64-darwin` (sandbox = true)
<details>
  <summary>:x: 14 packages failed to build:</summary>
  <ul>
    <li>apkg</li>
    <li>apkg.dist</li>
    <li>bdf2psf</li>
    <li>cargo-deb</li>
    <li>cozette</li>
    <li>debian-devscripts</li>
    <li>dpkg</li>
    <li>git-buildpackage</li>
    <li>git-buildpackage.dist</li>
    <li>gohufont</li>
    <li>perl538Packages.pacup</li>
    <li>perlPackages.pacup (perl540Packages.pacup)</li>
    <li>sslmate-agent</li>
    <li>uw-ttyp0</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>kochi-substitute</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `aarch64-darwin`</summary>
<details>
<summary>dpkg</summary>
<pre>
ERROR: 78 tests were run,
3 failed unexpectedly.
1 test was skipped.
## -------------------------- ##
## testsuite.log was created. ##
## -------------------------- ##

Please send &#x27;src/at/testsuite.log&#x27; and all information you think might help:

   To: &lt;debian-dpkg@lists.debian.org&gt;
   Subject: [dpkg 1.22.21] testsuite: 45 51 57 failed

You may investigate any problem if you feel able to do so, in which
case the test suite provides a good starting point.  Its output may
be found below &#x27;src/at/testsuite.dir&#x27;.

make[1]: *** [Makefile:1458: autotest-installcheck] Error 1
make[1]: Leaving directory &#x27;/nix/build/nix-8617-2631056047/dpkg/src&#x27;
make: *** [Makefile:769: installcheck-recursive] Error 1</pre>
</details>
</details>

--- a/pkgs/by-name/dp/dpkg/package.nix
+++ b/pkgs/by-name/dp/dpkg/package.nix
@@ -14,17 +14,30 @@
   autoreconfHook,
   pkg-config,
   diffutils,
+  versionCheckHook,
   glibc ? !stdenv.hostPlatform.isDarwin,
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "dpkg";
   version = "1.22.21";
 
   src = fetchgit {
     url = "https://git.launchpad.net/ubuntu/+source/dpkg";
-    rev = "applied/${version}";
-    hash = "sha256-UiXZfwvgsgyXR6olNzKelt/3Fgtp7KU8UbTRRkDl8wY=";
+    tag = "applied/${finalAttrs.version}";
+    leaveDotGit = true;
+    # Fix filename conflict on case-insensitive filesystems
+    postFetch = ''
+      pushd $out
+      git checkout HEAD -- scripts/t/Dpkg_BuildTree.t
+      mv scripts/t/Dpkg_BuildTree.t scripts/t/Dpkg_BuildTreeC.t
+      substituteInPlace scripts/Makefile.am --replace-fail t/Dpkg_BuildTree.t t/Dpkg_BuildTreeC.t
+      substituteInPlace scripts/Makefile.in --replace-fail t/Dpkg_BuildTree.t t/Dpkg_BuildTreeC.t
+      git checkout HEAD -- scripts/t/dpkg_buildtree.t
+      rm -rf .git
+      popd
+    '';
+    hash = "sha256-LK6nOPewjRyKyHdwJgmLILoZ6sEfJzRtC7pIeWz01lA=";
   };
 
   configureFlags = [
@@ -48,7 +61,7 @@ stdenv.mkDerivation rec {
     PATH=$TMPDIR:$PATH
 
     for i in $(find . -name Makefile.in); do
-      substituteInPlace $i --replace "install-data-local:" "disabled:" ;
+      substituteInPlace $i --replace-quiet "install-data-local:" "disabled:" ;
     done
 
     # Skip check broken when cross-compiling.
@@ -61,21 +74,26 @@ stdenv.mkDerivation rec {
 
     # Dpkg commands sometimes calls out to shell commands
     substituteInPlace lib/dpkg/dpkg.h \
-       --replace '"dpkg-deb"' \"$out/bin/dpkg-deb\" \
-       --replace '"dpkg-split"' \"$out/bin/dpkg-split\" \
-       --replace '"dpkg-query"' \"$out/bin/dpkg-query\" \
-       --replace '"dpkg-divert"' \"$out/bin/dpkg-divert\" \
-       --replace '"dpkg-statoverride"' \"$out/bin/dpkg-statoverride\" \
-       --replace '"dpkg-trigger"' \"$out/bin/dpkg-trigger\" \
-       --replace '"dpkg"' \"$out/bin/dpkg\" \
-       --replace '"debsig-verify"' \"$out/bin/debsig-verify\" \
-       --replace '"rm"' \"${coreutils}/bin/rm\" \
-       --replace '"cat"' \"${coreutils}/bin/cat\" \
-       --replace '"diff"' \"${diffutils}/bin/diff\"
+       --replace-fail '"dpkg-deb"' \"$out/bin/dpkg-deb\" \
+       --replace-fail '"dpkg-split"' \"$out/bin/dpkg-split\" \
+       --replace-fail '"dpkg-query"' \"$out/bin/dpkg-query\" \
+       --replace-fail '"dpkg-divert"' \"$out/bin/dpkg-divert\" \
+       --replace-fail '"dpkg-statoverride"' \"$out/bin/dpkg-statoverride\" \
+       --replace-fail '"dpkg-trigger"' \"$out/bin/dpkg-trigger\" \
+       --replace-fail '"dpkg"' \"$out/bin/dpkg\" \
+       --replace-fail '"debsig-verify"' \"$out/bin/debsig-verify\" \
+       --replace-fail '"rm"' \"${coreutils}/bin/rm\" \
+       --replace-fail '"cat"' \"${coreutils}/bin/cat\" \
+       --replace-fail '"diff"' \"${diffutils}/bin/diff\"
+  ''
+  + lib.optionalString stdenv.hostPlatform.isDarwin ''
+    # realpath("/var/lib/dpkg", NULL) gives EPERM on sandboxed darwin instead of the expected ENOENT,
+    # which makes some tests fail.
+    sed -i '/opts normalize/a AT_SKIP_IF([true])' src/at/chdir.at
   ''
   + lib.optionalString (!stdenv.hostPlatform.isDarwin) ''
     substituteInPlace src/main/help.c \
-       --replace '"ldconfig"' \"${glibc.bin}/bin/ldconfig\"
+       --replace-fail '"ldconfig"' \"${glibc.bin}/bin/ldconfig\"
   '';
 
   buildInputs = [
@@ -96,7 +114,7 @@ stdenv.mkDerivation rec {
   postInstall = ''
     for i in $out/bin/*; do
       if head -n 1 $i | grep -q perl; then
-        substituteInPlace $i --replace \
+        substituteInPlace $i --replace-fail \
           "${perl}/bin/perl" "${perl}/bin/perl -I $out/${perl.libPrefix}"
       fi
     done
@@ -105,6 +123,10 @@ stdenv.mkDerivation rec {
     cp -r scripts/t/origins $out/etc/dpkg
   '';
 
+  doInstallCheck = true;
+  nativeInstallCheckInputs = [ versionCheckHook ];
+  versionCheckProgramArg = "--version";
+
   setupHook = ./setup-hook.sh;
 
   meta = with lib; {
@@ -112,7 +134,7 @@ stdenv.mkDerivation rec {
     homepage = "https://wiki.debian.org/Teams/Dpkg";
     license = licenses.gpl2Plus;
     platforms = platforms.unix;
-    broken = stdenv.hostPlatform.isDarwin;
     maintainers = with maintainers; [ siriobalmelli ];
+    mainProgram = "dpkg";
   };
-}
+})

